src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -262,85 +157,6 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
-      .jsonPath("$.draftAdjudication.incidentDetails.handoverDeadline").isEqualTo("2010-11-14T10:00:00")
-      .jsonPath("$.draftAdjudication.incidentDetails.locationId").isEqualTo(3)
-  }
-
-  @Test
-  fun `edit offence details for the draft adjudication`() {
-    val testAdjudication = IntegrationTestData.ADJUDICATION_1
-    val intTestData = integrationTestData()
-    val intTestBuilder = IntegrationTestScenarioBuilder(intTestData, this)
-
-    val intTestScenario = intTestBuilder
-      .startDraft(testAdjudication)
-      .setApplicableRules()
-      .setIncidentRole()
-      .setOffenceData()
-
-    webTestClient.put()
-      .uri("/draft-adjudications/${intTestScenario.getDraftId()}/offence-details")
-      .headers(setHeaders())
-      .bodyValue(
-        mapOf(
-          "offenceDetails" to listOf(IntegrationTestData.BASIC_OFFENCE),
-        )
-      )
-      .exchange()
-      .expectStatus().isCreated
-      .expectBody()
-      .jsonPath("$.draftAdjudication.id").isNumber
-      .jsonPath("$.draftAdjudication.prisonerNumber").isEqualTo(testAdjudication.prisonerNumber)
-      .jsonPath("$.draftAdjudication.incidentDetails.dateTimeOfIncident")
-      .isEqualTo(testAdjudication.dateTimeOfIncidentISOString)
-      .jsonPath("$.draftAdjudication.incidentDetails.handoverDeadline")
-      .isEqualTo(testAdjudication.handoverDeadlineISOString)
-      .jsonPath("$.draftAdjudication.incidentDetails.locationId").isEqualTo(testAdjudication.locationId)
-      .jsonPath("$.draftAdjudication.incidentRole.roleCode").isEqualTo(testAdjudication.incidentRoleCode)
-      .jsonPath("$.draftAdjudication.incidentRole.offenceRule.paragraphNumber")
-      .isEqualTo(testAdjudication.incidentRoleParagraphNumber)
-      .jsonPath("$.draftAdjudication.incidentRole.offenceRule.paragraphDescription")
-      .isEqualTo(testAdjudication.incidentRoleParagraphDescription)
-      .jsonPath("$.draftAdjudication.incidentRole.associatedPrisonersNumber")
-      .isEqualTo(testAdjudication.incidentRoleAssociatedPrisonersNumber)
-      .jsonPath("$.draftAdjudication.offenceDetails[0].offenceCode")
-      .isEqualTo(IntegrationTestData.BASIC_OFFENCE.offenceCode)
-      .jsonPath("$.draftAdjudication.offenceDetails[0].offenceRule.paragraphNumber")
-      .isEqualTo(IntegrationTestData.BASIC_OFFENCE.paragraphNumber)
-      .jsonPath("$.draftAdjudication.offenceDetails[0].offenceRule.paragraphDescription")
-      .isEqualTo(IntegrationTestData.BASIC_OFFENCE.paragraphDescription)
-      .jsonPath("$.draftAdjudication.offenceDetails[0].victimPrisonersNumber").doesNotExist()
-      .jsonPath("$.draftAdjudication.offenceDetails[0].victimStaffUsername").doesNotExist()
-      .jsonPath("$.draftAdjudication.offenceDetails[0].victimOtherPersonsName").doesNotExist()
-      .jsonPath("$.draftAdjudication.offenceDetails[1]").doesNotExist()
-  }
-
-  @Test
-  fun `edit the incident statement`() {
```

Comment:
Sorry - wrong test....this one actually had a better name and actually edited the statement..

The one we have left tests something specific and isn't in keeping with the rest of the int tests. You could change the name of the other, but it really ought to test the statement been changed because that's the important bit.